radius_server/Dockerfile.freeradius

```diff
@@ -1,4 +1,10 @@
-FROM ubuntu:20.10
+FROM ubuntu:20.04
+
+ENV TZ=America/Los_Angeles
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt update
```

Comment:
for whatever reason (as in I forget why) it's customary to put the apt update in the same command (so using &&) -- see the one below

Reply:
Missed that. Updated.